gui/src/components/Layout.svelte

```diff
@@ -4,16 +4,32 @@ import IconButton from './IconButton.svelte';
 import * as router from 'svelte-spa-router';
 
 import Feedback from './mono/feedback.svelte';
+import GoBack from './mono/leftarrow.svelte';
+
+export let showBackButton: boolean = false;
```

Comment:
What's the purpose of having separate props for both showBackButton and backButtonRoute? Seems like you could make the default value of backButtonRoute undefined or empty string, and then you only need to specify that instead of both the route and the boolean flag.